spyder/plugins/application/container.py

```diff
@@ -331,3 +349,19 @@ def report_missing_dependencies(self):
             message_box.setWindowTitle(_('Error'))
             message_box.setText(message)
             message_box.show()
+
+    @Slot()
+    def restart_normal(self):
+        # always restart in standard mode
```

Comment:
```suggestion
        """Restart in standard mode."""
```

We can make this a docstring.